contracts/sol6/fprConversionRates/QtyStepConversionRates.sol

```diff
@@ -0,0 +1,436 @@
+pragma solidity 0.6.6;
+
+import "../IERC20.sol";
+import "../utils/Utils5.sol";
+import "../IConversionRates.sol";
+import "./SimpleVolumeImbalanceRecorder.sol";
+
+contract QtyStepConversionRates is IConversionRates, SimpleVolumeImbalanceRecorder, Utils5 {
+    int256 internal constant MAX_IMBALANCE = 2**255 - 1;
+    uint256 internal constant POW_2_128 = 2**128;
+    int256 internal constant MAX_BPS_ADJUSTMENT = 10**11; // 1B %
+    int256 internal constant MIN_BPS_ADJUSTMENT = -100 * 100; // cannot go down by more than 100%
+    // step data constant
+    uint256 internal constant MAX_STEPS_IN_FUNCTION = 10;
```

Comment:
we can have more steps here since removed imbalance steps
can use 13

in enhanced steps we used 16
but since then gas has become a bigger issue. So I suggest 13 as limit.
Any comments welcome

Reply:
Actually, the deployer can change this when compile and deploy.

Reply:
Deployer can change this and compile again. Which is also a way for us to hire on-chain code for conversionRate contract.

Reply:
OK
but lets use default of 13 here
please update it.

Reply:
change as request